src/tests/routes/auth.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+// import * as Factory from '../../helpers/factory';
+import db from '../../models';
+import app from '../../app';
+
+dotenv.config();
+
+chai.use(chaiHttp);
+chai.should();
+
+const userEmail = {
+  email: 'luctunechi45@gmail.com',
+};
+
+const notMatch = {
+  passwordOne: 'newpasswordi',
+  passwordTwo: 'newpassworda'
+};
+
+const newPassword = {
+  passwordOne: 'newpassworda',
+  passwordTwo: 'newpassworda',
+};
+
+const newUser = {
```

Comment:
I suggest you use the helper that I defined to generate fake user accounts

Reply:
there shouldn't be static declaration objects here, create files to generate this data or reuse what is already there.

Reply:
Changes applied